pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/secretmanagersecretversion/create.yaml

```diff
@@ -17,6 +17,7 @@ kind: SecretManagerSecretVersion
 metadata:
   annotations:
     cnrm.cloud.google.com/project-id: ${projectId}
+    alpha.cnrm.cloud.google.com/reconciler: direct
```

Comment:
Now that the controller for this test is switched to direct, looks you will need to re-record VCR (following https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/README.VCRTesting.md#record-mode). Or, maybe we can remove VCR coverage?

Reply:
I regenerated the VCR log